website/docs/r/pi_capture.html.markdown

```diff
@@ -0,0 +1,97 @@
+---
+
+subcategory: "Power Systems"
+layout: "ibm"
+page_title: "IBM: pi_capture"
+description: |-
+  Manages IBM Capture instance in the Power Virtual Server cloud.
+---
+
+# ibm_pi_capture
+Create or delete for Capture Power System Virtual Server Instance
+
+For more information, about IBM power virtual server cloud, see [getting started with IBM Power Systems Virtual Servers](https://cloud.ibm.com/docs/power-iaas?topic=power-iaas-getting-started).
+
+## Example usage
+The following example creates a Capture Power System Virtual Server Instance.
+
+```terraform
+
+resource "ibm_pi_capture" "test_capture  "{
```

Comment:
Remove additional spaces. 
Also format the examples with proper indentation for each line.

Reply:
Done